README.md

```diff
@@ -40,10 +40,10 @@ $ make test
 
 ### Updating plugins.json
 
-At the moment, the Marketplace simply points at the latest release of a fixed set of Mattermost plugins. In the future, this database will be fine-tuned to facilitate tracking multiple versions for the appropriate Mattermost server version. To update `plugins.json`, simply run:
+At the moment, the Marketplace simply points at the latest release of a fixed set of Mattermost plugins. In the future, this database will be fine-tuned to facilitate tracking multiple versions for the appropriate Mattermost server version. To update `plugins.json`, ensure you have [jq](https://stedolan.github.io/jq/) and [sponge](https://linux.die.net/man/1/sponge) installed and run:
 
 ```
-$ go run ./cmd/generator --github-token <your github token> --debug > plugins.json
+$ go run ./cmd/generator --github-token <your github token> --existing plugins.json --debug | jq | sponge plugins.json
```

Comment:
Maybe we add this as a `Makefile` target for now? I'm surely will come here every time to copy and past it.